digital-land/digital-land-python
# Feat/exp 3

## What type of PR is this? (check all applicable)


## Description

Alter and improve the expectations work. It has become clear that the inputs and outputs were too complicated to take in and use. this introduces a more simplified out put and input based on configuration files.

## Related Tickets & Documents


- Ticket Link
- Related Issue #
- Closes #

## QA Instructions, Screenshots, Recordings

Tests should replicate al the relevant behaviour so worth reviewing them to ensure no other cases are needed. 

this will be supported by changes to makerules, will update with a PR link

this will be supported by a branch of collection task which can be ran locally to check. This won't need a PR but will set up relevant environment variables to easily run.

_Please replace this line with instructions on how to test your changes, a note
on the devices and browsers this has been tested on, as well as any relevant
images for UI changes._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] Are there any dependencies on other PRs or Work?

Will need a demonstration to our team, DM team and finally to Providers
